x-pack/plugins/secops/public/components/page/hosts/authentications_table/index.tsx

```diff
@@ -146,30 +147,66 @@ const getAuthenticationColumns = (startDate: number) => [
     hideForMobile: false,
     render: ({ node }: AuthenticationsEdges) => <>{defaultToEmpty(node.failures)}</>,
   },
+  {
+    name: i18n.LAST_FAILED_TIME,
+    truncateText: false,
+    hideForMobile: false,
+    render: ({ node }: AuthenticationsEdges) => {
+      return (
+        <>
+          {has('lastFailure.timestamp', node) ? (
+            <FormattedRelative value={new Date(node.lastFailure!.timestamp!)} />
+          ) : (
+            getEmptyValue()
+          )}
+        </>
+      );
+    },
+  },
+  {
+    name: i18n.LAST_FAILED_SOURCE,
+    truncateText: false,
+    hideForMobile: false,
+    render: ({ node }: AuthenticationsEdges) => <>{getOrEmpty('lastFailure.source.ip', node)}</>,
+  },
+  {
+    name: i18n.LAST_FAILED_DESTINATION,
+    truncateText: false,
+    hideForMobile: false,
+    render: ({ node }: AuthenticationsEdges) => <>{getOrEmpty('lastFailure.host.name', node)}</>,
+  },
   {
     name: i18n.SUCCESSES,
     truncateText: false,
     hideForMobile: false,
     render: ({ node }: AuthenticationsEdges) => <>{defaultToEmpty(node.successes)}</>,
   },
   {
-    name: i18n.FROM,
+    name: i18n.LAST_SUCCESSFUL_TIME,
     truncateText: false,
     hideForMobile: false,
-    render: ({ node }: AuthenticationsEdges) => <>{defaultToEmpty(node.source.ip)}</>,
+    render: ({ node }: AuthenticationsEdges) => {
+      return (
+        <>
+          {has('lastSuccess.timestamp', node) ? (
+            <FormattedRelative value={new Date(node.lastSuccess!.timestamp!)} />
+          ) : (
+            getEmptyValue()
+          )}
+        </>
+      );
+    },
   },
   {
-    name: i18n.TO,
+    name: i18n.LAST_SUCCESSFUL_SOURCE,
     truncateText: false,
     hideForMobile: false,
-    render: ({ node }: AuthenticationsEdges) => <>{defaultToEmpty(node.host.name)}</>,
+    render: ({ node }: AuthenticationsEdges) => <>{getOrEmpty('lastSuccess.source.ip', node)}</>,
   },
   {
-    name: i18n.LATEST,
+    name: i18n.LAST_SUCCESSFUL_DESTINATION,
     truncateText: false,
     hideForMobile: false,
-    render: ({ node }: AuthenticationsEdges) => (
-      <>{node.latest ? <FormattedRelative value={new Date(node.latest)} /> : getEmptyValue()}</>
-    ),
+    render: ({ node }: AuthenticationsEdges) => <>{getOrEmpty('lastSuccess.host.name', node)}</>,
```

Comment:
feature request (does not have to be in this PR): this column should be draggable to the timeline